.husky/commit-msg

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn commitlint --edit 
+# yarn commitlint --edit 
```

Comment:
That's probably a leftover?

Reply:
hmmm... nothing I recall to have changed. Will restore and commit original state